### ISSUE TITLE
IEC 80000-13: equation in table cell shows equation label outside of cell

### ISSUE THREAD

**Issue by ronaldtse** [metanorma/metanorma-iec#248]
Original issues:
* https://github.com/metanorma/iso-iec-80000/pull/47
* https://github.com/metanorma/iso-iec-80000/pull/48

<img width="534" alt="Screenshot 2023-06-06 at 10 53 43 AM" src="https://github.com/metanorma/metanorma-iec/assets/11865/889edb2a-59ce-44b0-8f3c-c9299111a48d">

**Comment by Intelligent2013** [comment]
@ronaldtse I can't repeat the issue:
![image](https://github.com/metanorma/metanorma-iec/assets/5559926/8152a4a7-a909-4449-b0f4-1914392a2079)

On your screenshot there isn't blue color for links and font looks as 'Arial', not 'Cambria'.


Found another case:
![image](https://github.com/metanorma/metanorma-iec/assets/5559926/aef4d24b-bae4-4c00-b60d-56e97c0488bb)

**Comment by Intelligent2013** [comment]
@ronaldtse I've run ISO flavor. For IEC I can repeat the issue. Thanks.

**Comment by Intelligent2013** [comment]
Issue occurs due the negative right margin. Formula number in the text renders on the right page margin:
```xml
</fo:table-cell>
<fo:table-cell display-align="center">
	<fo:block text-align="right" margin-right="-10mm">(15)</fo:block>
</fo:table-cell>
</fo:table-row>
</fo:table-body>
</fo:table>
```

If formula inside in the table, then no need the right margin.

**Comment by Intelligent2013** [comment]
Fixed:
![image](https://github.com/metanorma/metanorma-iec/assets/5559926/547bcfe2-281f-4afc-9ee7-823bcde94aef)


Also fixed the issue (in mn2pdf v1.72):
![image](https://github.com/metanorma/metanorma-iec/assets/5559926/368451ec-461d-4562-9445-8d2aea6274f8)

Now:
![image](https://github.com/metanorma/metanorma-iec/assets/5559926/fb67bb02-7bcc-48eb-8d2d-ed63599fff83)

**Comment by ronaldtse** [comment]
Thanks @Intelligent2013 !
